packages/cli/src/commands/verify.ts

```diff
@@ -0,0 +1,84 @@
+import type { CommandModule } from "yargs";
+import { verify } from "../verify";
+import { logError } from "../utils/errors";
+import { loadConfig } from "@latticexyz/config/node";
+import { WorldConfig, resolveWorldConfig } from "@latticexyz/world/internal";
+import { worldToV1 } from "@latticexyz/world/config/v2";
+import { getOutDirectory, getSrcDirectory } from "@latticexyz/common/foundry";
+import { getExistingContracts } from "../utils/getExistingContracts";
+import { getContractData } from "../utils/getContractData";
+import { defaultModuleContracts } from "../utils/defaultModuleContracts";
+import { Hex } from "viem";
+
+type Options = {
+  worldAddress?: string;
+  configPath?: string;
+  profile?: string;
+  srcDir?: string;
+};
+
+const commandModule: CommandModule<Options, Options> = {
+  command: "verify",
+
+  describe: "Verify contracts",
+
+  builder(yargs) {
+    return yargs.options({
+      configPath: { type: "string", desc: "Path to the config file" },
```

Comment:
is this the mud config?
```suggestion
      configPath: { type: "string", desc: "Path to the mud.config.ts file" },
```

Reply:
I put "Path to the MUD config file" as it felt less filename/implementation-specific